my_enum.rb

```diff
@@ -0,0 +1,17 @@
+module MyEnumerable
+  def all?(&block)
+    each { |item| return false unless block.call(item) }
+    true
+  end
+
+  def filter(&block)
+    res = []
+    each { |item| result << item if block.call(item) }
+    res
+  end
```

Comment:
The test for the filter method is throwing an error due to the undefined local variable `result` present in line 9 as shown below. :point_down: Kindly check and correct this error. I believe it should be `res` instead of `result`.


![Screenshot from 2023-03-04 10-25-18](https://user-images.githubusercontent.com/65233860/222894631-bd0f1548-fd1c-45e7-bd01-0e1d72bd9134.png)